monkestation/code/modules/surgery/organs/tails.dm

```diff
@@ -0,0 +1,60 @@
+/obj/item/organ/tail/monkey
+	name = "monkey tail"
+	desc = "A severed monkey tail. If found, please call 1-800-OOK-EEEK"
+	icon = 'monkestation/icons/mob/mutant_bodyparts.dmi'
+	icon_state = "monkey"
+	color = "#5f4637" //for debugging
```

Comment:
Debugging still needed?

Reply:
Ehh this just colors the dropped tail icon, can probably stay there. can remove the comment tho